examples/combobox/combobox-autocomplete-both.html

```diff
@@ -196,7 +191,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
         <strong>NOTE:</strong> When visual focus is in the listbox, DOM focus remains on the textbox and the value of <code>aria-activedescendant</code> on the textbox is set to a value that refers to the listbox option that is visually indicated as focused.
```

Comment:
For Up arrow, change "last" to "previous" as follows:
> If the listbox is displayed and a suggestion is selected, moves visual focus to the last suggested value.

to:
> If the listbox is displayed and a suggestion is selected, moves visual focus to the previous suggested value.

Reply:
That's not what happens when focus is in the textbox ... this table is for textbox. If it is in the textbox, up takes you to last suggestion

Reply:
Nope; with focus in textbox, up arrow goes to previous suggestion in list. :)
Steps:
- focus textbox
- type down arrow 6 times; you will be on California
- type up arrow once; you will be on Arkansas, not Wyoming